leader-election-impl/src/main/java/com/palantir/leader/PaxosLeaderElectionService.java

```diff
@@ -265,11 +238,6 @@ private boolean isThisNodeTheLeaderFor(PaxosValue value) {
         return value.getLeaderUUID().equals(proposer.getUuid());
     }
 
-    // This is used by an internal product CLI.
-    public ImmutableList<PaxosAcceptor> getAcceptors() {
-        return acceptors;
-    }
-
```

Comment:
To confirm: We've since changed the internal product CLI in question, I'm guessing?

Reply:
yeah, I've made most of the migrations, it's just the builder parts that will break.